srcpkgs/bacon/template

```diff
@@ -12,10 +12,15 @@ license="MIT"
 homepage="https://www.basic-converter.org"
 changelog="https://www.basic-converter.org/stable/CHANGES"
 distfiles="http://www.basic-converter.org/stable/bacon-${version}.tar.gz"
-checksum=96bbfeebb506018a6e208dc85e6daded49778ec8b4d037854bda721af890e501
+checksum=dbd7c983530743922c8ae21ef5c471df894ece9af12b4075564deb7e0659babf
 disable_parallel_build=yes
 nocross=yes
 
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	makedepends+=" musl-fts-devel"
+	export LDLIBS="-lfts"
```

Comment:
```suggestion
	LDFLAGS="-lfts"
```

This is one ugly build system lol

Reply:
It seems to have passed.
Though I'm worried that it might fail to build later on.

Reply:
If you can tell me some simple steps to use it, I can test locally on musl.

Reply:
> If you can tell me some simple steps to use it, I can test locally on musl.

What I meant is that I'm worried that it might fail to build after being merged. As I tried to build it in a VM and it ended up with a segfault.

Using it is simple enough. Just run `bacon` (or `bacon.sh`?) with the file to make. e.g. `bacon hello.bac`
Their website has a bunch of [example](http://basic-converter.org/#examples) code.

Reply:
Just got a segfault now while trying to build the package D:

You shouldn't need a VM to test things on musl. Just `./xbps-src -m masterdir-musl binary-bootstrap x86_64-musl` is enough to create a musl masterdir.

Reply:
Well, if it segfaults during the build we can just try again, I have no idea where to even start with investigating it.